rskj-core/src/main/java/org/ethereum/datasource/RocksDbDataSource.java

```diff
@@ -86,13 +91,6 @@ public void init() {
 
             Objects.requireNonNull(name, "no name set to the db");
 
-            options.setCreateIfMissing(true);
-            options.setCompressionType(CompressionType.NO_COMPRESSION);
-            options.setArenaBlockSize(GENERAL_SIZE);
-            options.setWriteBufferSize(GENERAL_SIZE);
-
-            options.setParanoidChecks(true);
-
```

Comment:
probably it's no need to pass `options` into the `openDb()` method - it cap be used as a class field directly, wdyt?

Reply:
yep, you are absolutely right